### ISSUE TITLE
Ray component: Core - pypi repo

### ISSUE THREAD

**Issue by eric-anderson** [ray-project/ray#46104]
### What happened + What you expected to happen

We build docker images using ray for linux images on both x86_64 and aarch64/arm64 (for mac clients).
This worked with ray 2.10 but did not work when we upgraded to ray 2.24.  If I attempt to explicitly install it, I get:
```
% poetry run pip install 'ray==2.24.0'
ERROR: Could not find a version that satisfies the requirement ray==2.24.0 (from versions: 2.3.0, 2.3.1, 2.4.0, 2.5.0, 2.5.1, 2.6.0, 2.6.1, 2.6.2, 2.6.3, 2.7.0rc0, 2.7.0, 2.7.1, 2.7.2, 2.8.0, 2.8.1, 2.9.0, 2.9.1, 2.9.2, 2.9.3, 2.10.0, 2.20.0)
ERROR: No matching distribution found for ray==2.24.0
```

If I check the pypi files for 2.10.0 https://pypi.org/project/ray/2.10.0/#files I find a bunch of *manylinux*aarch64* files, e.g. 
ray-2.10.0-cp311-cp311-manylinux2014_aarch64.whl 
However, for 2.24.0 https://pypi.org/project/ray/2.24.0/#files, those files are not present. there are manylinux/arm64 files and macos/arm64 files but no manylinux/aarch64 files. 
The files were last present for 2.20.0 https://pypi.org/project/ray/2.20.0/#files

Are these files supposed to be present?  The installation page says that aarch64 is supported. https://docs.ray.io/en/latest/ray-overview/installation.html

### Versions / Dependencies

```
% poetry run pip --version
pip 23.0.1 from /home/ci/.cache/pypoetry/virtualenvs/sycamore-monorepo-ip07jmFR-py3.11/lib/python3.11/site-packages/pip (python 3.11)
% poetry run python --version
Python 3.11.2
% uname -a
Linux ip-172-31-19-76 6.1.0-21-cloud-arm64 #1 SMP Debian 6.1.90-1 (2024-05-03) aarch64 GNU/Linux
% cat /etc/debian_version 
12.5
```

### Reproduction script

On a linux/arm machine, for example, a t4g.xlarge in AWS, with poetry and pip installed:
```
% pip install 'ray==2.24.0'
# or
% poetry run pip install 'ray==2.24.0'
```

### Issue Severity

Medium: It is a significant difficulty but I can work around it.

**Comment by aslonnie** [comment]
we did not release linux arm64 for 2.24

**Comment by aslonnie** [comment]
we are planning to do full releases in the future though.

you can also use 2.30, if it works for you.

I don't think we will fix this (backfill) particularly for 2.24 though.

**Comment by eric-anderson** [comment]
Tested on our linux/arm64 machine and verified I can install 2.30.  We're fine staying on 2.20 until we can skip forward, so no need to backfill.

Closing bug.
